cls/_zpkg/isc/sc/git/Favorites.cls

```diff
@@ -0,0 +1,86 @@
+Class %zpkg.isc.sc.git.Favorites
+{
+    ClassMethod ConfigureFavoriteNamespaces(username As %String, newNamespaces As %Library.DynamicObject)
+{
+    // Convert to $listbuild
+    set namespaces = $lb("")
```

Comment:
Do you intend to have an empty option? An empty $listbuild list is just the empty string.